src/components/themed/TransactionListRow.tsx

```diff
@@ -106,7 +105,7 @@ export function TransactionListRow(props: Props) {
       fiatSymbol={getSymbolFromCurrency(nonIsoFiatCurrencyCode)}
       onPress={handlePress}
       onLongPress={handleLongPress}
-      isSentTransaction={isSentTransaction(transaction)}
+      isSentTransaction={transaction.isSend}
```

Comment:
Let's do this differently:

1. Remove the `isSentTransaction` flag, since we already pass the full `transaction` itself to the row.
2. Inside the row, continue to use `> 0` for deciding whether to show the `-` sign or not.
3. Inside the row, pass `>= 0 ? 'income' : 'expense'` as the second parameter to `splitCategory`, to pick the right default category.